### PR TITLE
Capture output to silence verbose codesign output.

### PR DESCRIPTION
**Pull request by erykoff**

### CODE REVIEW

**Comment by isuruf** [comment]
@cla-bot check

**Comment by cjmartian** [comment]
@anaconda-issue-bot check

**Review by chenghlee — APPROVED** [review]
Never mind; since this is for osx-arm64 only, guarded by an appropriate conditional, and we don't plan to support Py3.6 for that platform, use of `capture_output` shouldn't be an issue.

File: conda/core/portability.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def _update_prefix(original_data):
 
     if updated and mode == FileMode.binary and subdir == "osx-arm64" and sys.platform == "darwin":
         # Apple arm64 needs signed executables
-        subprocess.run(['/usr/bin/codesign', '-s', '-', '-f', realpath(path)])
+        subprocess.run(['/usr/bin/codesign', '-s', '-', '-f', realpath(path)], capture_output=True)
```
[the remark this answers is not in the document]
Comment (erykoff) [reply conda/core/portability.py:67]: FYI, for whatever reason `stdout=subprocess.DEVNULL` did not seem to work.  But thankfully it is moot for osx-arm64.

**Review by cjmartian — APPROVED** [review]

**Pull request merged**